.github/workflows/node.js.yml

```diff
@@ -0,0 +1,30 @@
+name: Angular build
+
+on:
+  push:
+    branches: [ 'main' ]
+  pull_request:
+    branches: [ 'main' ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [18.x, 20.x, 22.x]
```

Comment:
Let's build and maitanance only one specific node version. During new features implementation there might be some issues for older node rather than the new one.
I would like to recommend support only node 22.

Reply:
Done 👍